### PR TITLE
Fix enum discriminant

### PR DESCRIPTION
**Pull request by echeran**
Typo fix for #2045

### CODE REVIEW

**Review by sffc — DISMISSED** [review]
Good to get this in

**Comment by sffc** [comment]
(but wait for review from @Manishearth )

**Review by Manishearth — DISMISSED** [review]
oh right i was going to fix this

**Review by sffc — APPROVED** [review]

**Pull request merged**
